python/python/tests/test_async_client.py

```diff
@@ -5559,6 +5559,49 @@ async def test_getex(self, redis_client: TRedisClient):
         )
         assert await redis_client.ttl(key1) == -1
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_lolwut(self, redis_client: TRedisClient):
+        result = await redis_client.lolwut()
+        assert "Redis ver. " in result
+        result = await redis_client.lolwut(parameters=[50, 20])
+        assert "Redis ver. " in result
+        result = await redis_client.lolwut(6)
+        assert "Redis ver. " in result
+        result = await redis_client.lolwut(5, [30, 4, 4])
+        assert "Redis ver. " in result
+
+        if isinstance(redis_client, RedisClusterClient):
+            # test with multi-node route
+            result = await redis_client.lolwut(route=AllNodes())
+            if isinstance(result, dict):
+                for node_result in result.values():
+                    assert "Redis ver. " in node_result
+            else:
+                assert "Redis ver. " in result
+
+            result = await redis_client.lolwut(parameters=[10, 20], route=AllNodes())
+            if isinstance(result, dict):
+                for node_result in result.values():
+                    assert "Redis ver. " in node_result
+            else:
+                assert "Redis ver. " in result
+
+            # # test with single-node route
+            result = await redis_client.lolwut(2, route=RandomNode())
+            if isinstance(result, dict):
+                for node_result in result.values():
+                    assert "Redis ver. " in node_result
+            else:
+                assert "Redis ver. " in result
+
+            result = await redis_client.lolwut(2, [10, 20], RandomNode())
+            if isinstance(result, dict):
+                for node_result in result.values():
+                    assert "Redis ver. " in node_result
+            else:
+                assert "Redis ver. " in result
```

Comment:
I suggest altering this a bit since we know whether we should get back a dict or string based on the route we gave
```suggestion
            assert isinstance(result, dict)
            for node_result in result.values():
                assert "Redis ver. " in node_result

            result = await redis_client.lolwut(parameters=[10, 20], route=AllNodes())
            assert isinstance(result, dict):
            for node_result in result.values():
                assert "Redis ver. " in node_result

            # test with single-node route
            result = await redis_client.lolwut(2, route=RandomNode())
            assert "Redis ver. " in result

            result = await redis_client.lolwut(2, [10, 20], RandomNode())
            assert "Redis ver. " in result
```